src/dynamic_bridge.cpp

```diff
@@ -784,11 +787,11 @@ int main(int argc, char * argv[])
 
 
   // ROS 1 asynchronous spinner
-  ros::AsyncSpinner async_spinner(1);
+  ros::AsyncSpinner async_spinner(0, &ros1_callback_queue);
```

Comment:
this will use the all cores in the system to spin. i think using Single or Multi threaded executor is dependent on the use case. for example, someone just bridge the certain topic only, in that case we do not need Multi threaded executor.

i think at least we would want to provide the option for user, then user can decide what they need.

Reply:
OK, I'll introduce an option, and also append an argument (bool multithread = false) to some methods to check if it's necessary to create a custom `callbackgroup` or not.